src/dmd/cli.d

```diff
@@ -314,7 +314,42 @@ dmd -cov -unittest myprog.d
             "same as -i=-std,-core,-etc,-object"
         ),
         Option("i=[-]<pattern>,[-]<pattern>,...",
-            "include/exclude imported modules whose name matches one of <pattern>"
+            "include/exclude imported modules whose name matches one of <pattern>",
+            `this option enables "include import" mode, where the compiler will
+             include imported modules in the compilation, as if they were given
+             on the command line.
+
+             Determining which modules to include is based on a set of "module patterns".
+             These patterns are optionally included after the "-i=" option as a comma
```

Comment:
How about using a [delimited string](https://dlang.org/spec/lex.html#delimited_strings) and use back-ticks for command line things like `-i=`?